skimage/measure/profile.py

```diff
@@ -86,14 +88,24 @@ def profile_line(image, src, dst, linewidth=1,
            [1.        , 1.        , 0.        ],
            [1.41421356, 1.41421356, 0.        ]])
     """
+    if order is None:
+        order = 0 if image.dtype == bool else 1
+
+    if image.dtype == bool and order != 0:
+        warn("Input image dtype is bool. Interpolation is not defined "
```

Comment:
Why not just raise? I'm affraid that setting this precedent means that we will have to support this kind of operation almost indefinitely greatly increasing the scope of the API

Reply:
> I am not against raising but it is a little bit aggressive. May be can we then start a deprecation cycle specifying that starting from version 0.19 an error will be raised in this case.?